### ISSUE TITLE
Compatability with Windows recursive checking

### ISSUE THREAD

**Issue by ghost** [nabijaczleweli/checksums#27]
When i use `-r` flag it makes in format `text/file - hash`
On windows it should be `text\file - hash`

**Comment by nabijaczleweli** [comment]
What exactly is your point or use-case here? Windows accepts both / and \ as a path separator.

**Comment by ghost** [comment]
No, the problem here is that some programs accept only backlash format only.
There's a library for checking hashsums which is called IShash.dll
This library checks if everything is valid, and unfortunately enough it can read on backlash hashes.
